cfme/control/explorer.py

```diff
@@ -1226,6 +1228,10 @@ def _fill(self):
                 raise TypeError("evaluate must be either string with dropdown value or function "
                                 "or a 2-tuple with (dropdown value, dict of values to set)")
         fill(self.form, fill_details)
+        if form_func is None:
```

Comment:
instead of declaring the missing function and then calling it - just dont call it, if it is missing